docs/DeveloperGuide.md

```diff
@@ -302,11 +441,131 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. The command given is in an invalid format
+
+    * 3a1. ContaX shows an error message indicating that the given command is of invalid format.
+
+    * Use case resumes at step 2.
+
+**Use case: Edit an appointment**
+
+**MSS**
+
+1. User requests to list appointments
+2. ContaX shows a list of appointments
+3. User requests to modify a specific appointment and enters new values
+4. ContaX edits the appointment.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The command given is in an invalid format
+
+    * 3a1. ContaX shows an error message indicating that the given command is of invalid format.
+
+    * Use case resumes at step 2.
+    
+* 3b. The parameter given is in an invalid format
+
+    * 3b1. ContaX shows an error message indicating that the given parameter is of invalid format. 
+
+    * Use case resumes at step 2.
+
+**Use case: Export address book**
+
+**MSS**
+
+1. User requests to export address book as file for ContaX
+2. ContaX saves a CSV file to the disk according to the format requested by the user
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. User requests to export address book as file for Google Contacts
+   
+  Use case resumes at step 2.
+    
+* 1b. User requests to export address book as file for Microsoft Outlook
+
+  Use case resumes at step 2. 
+
+**Use Case: Import CSV file**
+
+**MSS**
+
+1. User requests to import a CSV file
+2. ContactX imports the given CSV file
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. Invalid CSV file selected
+
+    * 1a1. ContaX shows an error message indicating that the CSV file selected is invalid. 
+
+    * Use case ends.
```

Comment:
The numbering here will be affected if the above change is adopted.

Reply:
Noted